h/schemas/api/moderation.py

```diff
@@ -0,0 +1,36 @@
+import colander
+
+from h.models.annotation import ModerationStatus
+
+
+class ModerationStatusNode(colander.SchemaNode):
+    schema_type = colander.String
+    validator = colander.OneOf(["APPROVED", "PENDING", "DENIED", "SPAM"])
+
+    def deserialize(self, cstruct):
+        return ModerationStatus(cstruct) if cstruct else None
+
+
+class ChangeAnnotationModerationStatusSchema(colander.Schema):
+    """Schema for validating change-annotation-moderation-status API data."""
+
+    moderation_status = ModerationStatusNode()
+    annotation_updated = colander.SchemaNode(
+        colander.DateTime(),
+        description="Sentinel value to avoid conflicting updates. It should be match the current value of annotation.updated",
+    )
```

Comment:
I wonder if `annotation_updated` should be optional, and if it's provided then it much be correct? That way our own client and frontend code can use `annotation_updated` to make sure it never moderates based on an outdated version of an annotation. But other API users don't have to bother with this if they don't want it.

Reply:
Not sure, you likely have the annotation object around before moderating to passing the date is not a big ask for any client.